e2e/rbd.go

```diff
@@ -115,6 +115,15 @@ var _ = Describe("RBD", func() {
 			By("create a PVC and Bind it to an app with normal user", func() {
 				validateNormalUserPVCAccess(pvcPath, f)
 			})
+
+			By("create a PVC and Bind it to an app with ext4 as the FS ", func() {
+				deleteResource(rbdExamplePath + "storageclass.yaml")
+				createRBDStorageClass(f.ClientSet, f, map[string]string{"csi.storage.k8s.io/fstype": "ext4"})
+				validatePVCAndAppBindingExt4Mount(pvcPath, appPath, f)
```

Comment:
validatePVCAndAppBindingExt4Mount is not needed please reuse `validatePVCAndAppBinding`